src/types.rs

```diff
@@ -125,6 +125,7 @@ pub enum BinOper {
     Mul,
     Div,
     As,
+    Escape,
```

Comment:
@tyt2y3 

I think `LIKE ... ESCAPE ...` is not `BinOper`. If we locking into sqlite doc, for example, we can see that: Like Expression is special case. And so I move it to `SimpleExpr`. `LIKE` Can take only `String` (I cannot find other example).

Reply:
It should definitely not be value as it doesn't need to be escaped with a `?`. Unless I am mistaken it will be in the current implementation.

Reply:
Sorry for that. I just thought maintaining a SimpleExpr variant is significantly more cost than maintaining an additional BinOper.

Reply:
And that `BETWEEN A AND B` is also a quirky thing in SQL (that has some special treatment in our code generator)